examples/ad7124_DataCapture.m

```diff
@@ -0,0 +1,24 @@
+%% Script for capturing and displaying a continuous set of samples from a 
+%% connected AD7124-4 board
+
+% Instantiate the system object
+rx = adi.AD7124_8.Rx('uri','ip:analog.local');
+
+% Samples per frame cannot exceed 500 if all 16 channels need to be captured on ad7124-8 
+rx.SamplesPerFrame = 500; 
+rx.EnabledChannels = [1 2 3 4 5 6 7 8];
```

Comment:
Should we enable all 16 channels if the chosen device is AD7124-8?

Reply:
I think the example script should be catered to ad7124-4 as it is default device on firmware also. I would change the example script accordingly.